lib/game/widget_builder.dart

```diff
@@ -68,19 +85,18 @@ Widget _applyAdvancedGesturesDetectors(Game game, Widget child) {
     );
   }
 
-  if (game is MultiTouchDragDetector) {
+  if (_dragHandlers.isNotEmpty) {
     gestures[ImmediateMultiDragGestureRecognizer] =
         GestureRecognizerFactoryWithHandlers<
             ImmediateMultiDragGestureRecognizer>(
       () => ImmediateMultiDragGestureRecognizer(),
-      (ImmediateMultiDragGestureRecognizer instance) {
+      (MultiDragGestureRecognizer instance) {
         instance
           ..onStart = (Offset o) {
             final drag = DragEvent();
+            // TODO: Remove any padding/margin on the widget here
             drag.initialPosition = o;
```

Comment:
This is a problem, `initialPosition` here needs to take into consideration that there is padding outside of the flame widget, but since there is no local position here does anyone have a clue how to do that?

Reply:
I believe @renancaraujo 's PR will allow for precisely that!